src/pm/hydra/ui/mpich/utils.c

```diff
@@ -1635,6 +1635,18 @@ static HYD_status parse_args(char **t_argv)
     goto fn_exit;
 }
 
+void HYD_check_envs(void)
+{
+    int val;
+    if(MPL_env2int("MPIEXEC_PREFIX_DEFAULT", &val))
+    {
+        if(val > 0)
+	{
+	    prepend_rank_fn("prepend-rank", NULL);
+	}
+    }
+}
+
```

Comment:
The internal function should be declared as `static` and you don't need a namespace for static functions.